file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/sender/AlfrescoFileSender.java

```diff
@@ -29,21 +32,23 @@
     String alfrescoUploadUser;
 
     @Inject
-    ObjectMapper objectMapper;
-
-    @Inject
     AlfrescoRestClient restClient;
 
-
     @Override
     public FileData send(final String fileName, final byte[] content) {
-        final MultivaluedHashMap<String, Object> headers = new MultivaluedHashMap<>();
-        headers.put(ID, Collections.singletonList(alfrescoUploadUser));
-
-        restClient.post(alfrescoUploadPath, MULTIPART_FORM_DATA_TYPE, headers, entity(content, MULTIPART_FORM_DATA_TYPE));
-
-        //TODO: build response
-
-        return null;
+        try {
+            final Response response = restClient.post(alfrescoUploadPath, MULTIPART_FORM_DATA_TYPE, headersWithUserId(alfrescoUploadUser), entity(content, MULTIPART_FORM_DATA_TYPE));
+            final String responseEntity = response.readEntity(String.class);
+            if (response.getStatusInfo() != OK || isEmpty(responseEntity)) {
```

Comment:
maybe add a couple of blank lines here for formatting, to separate request, check etc